saleor/core/jwt_manager.py

```diff
@@ -69,6 +69,7 @@ def get_issuer(cls) -> str:
 
 class JWTManager(JWTManagerBase):
```

Comment:
Why do we extend class that is abstract / has no implementations? Can we replace it with an interface?

Reply:
This is surely out of the scope of this change?

Reply:
Yes, but I use conversation to learn / cleanup after